src/main/java/org/ohdsi/circe/cohortdefinition/CohortExpressionQueryBuilder.java

```diff
@@ -490,15 +490,21 @@ public String getCriteriaGroupQuery(CriteriaGroup group, String eventTable) {
       {
         if (group.type.toUpperCase().endsWith("LEAST"))
           occurrenceCountClause += ">= " + group.count;
-        else
+				else {
```

Comment:
I'd suggest to either use brackets in all parts of `if` or in none - from a codestyle perspective and easy-to-read perspective.

Reply:
Ok, will do.